databricks/koalas/missing/series.py

```diff
@@ -145,7 +137,6 @@ class _MissingPandasLikeSeries(object):
     rmod = unsupported_function('rmod')
     rolling = unsupported_function('rolling')
     round = unsupported_function('round')
-    rpow = unsupported_function('rpow')
```

Comment:
ditto.

Reply:
`rpow` is still being removed.

Reply:
Sorry, I initially implement both too and decided to take out right before opening a PR .. :-) ..